src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -92,6 +92,15 @@
                 &::before {
                     transform: scaleY(1);
                 }
+
+                .bcs-AnnotationActivity {
+                    &.is-interactive {
+                        &:hover,
+                        &:focus {
+                            animation-iteration-count: 0;
```

Comment:
The relationship between this component, its parent, and the animation isn't really clear. It seems like we may need to improve our component hierarchy at some point, but can we clarify things for now?